### ISSUE TITLE
Bad news - we did not collect dwi fieldmaps in the AP direction

### ISSUE THREAD

**Issue by oesteban** [TheAxonLab/hcph-sops#405]
While this problem does not justify repeating the data collection, unfortunately, I did not set the correct AP phase encoding for the AP fieldmaps of every session. When running the protocol, we trusted I had done it correctly, and we didn't check a single time.

I hope some of the pilot session have the four fieldmaps properly. I have checked and the functional fieldmaps are okay (although they are much lower resolution).

:(

**Comment by acionca** [comment]
Are you talking about the `fmap-epi_acq-b0` sequence or the `fmap-phasediff__gre` ?

**Comment by oesteban** [comment]
`fmap-epi_acq-b0_dir-AP` are all PA :(

**Comment by acionca** [comment]
It is weird, the protocol [summary pdf](https://www.axonlab.org/hcph-sops/assets/files/HCPh_RL.pdf) (for `HCPh_RL`) says the opposite (AP is AP but PA is also AP)

**Comment by oesteban** [comment]
True, I mistakenly reversed the direction: all `fmap-epi_acq-b0_dir-PA` are AP
